packages/node_modules/@ciscospark/react-container-message-composer/src/container.js

```diff
@@ -103,49 +115,128 @@ export class MessageComposer extends Component {
     props.setUserTyping(false, conversation, sparkInstance);
   }
 
+  @autobind
   handleTextAreaFocus() {
     const props = this.props;
     props.focusTextArea();
   }
 
+  @autobind
+  searchForMention(term) {
+    const {
+      avatar,
+      conversation,
+      user
+    } = this.props;
+
+    const participants = [];
+    conversation.get(`participants`).toJS().forEach((p) => {
+      if (user.get(`currentUser`).id !== p.id) {
+        participants.push({
+          display: p.displayName,
+          id: p.id,
+          avatarUrl: avatar.getIn([`items`, p.id])
+        });
+      }
+    });
+
+    if (term) {
+      return participants.filter((p) => p.display.toLowerCase().includes(term.toLowerCase()));
+    }
+    return participants;
+  }
+
+  @autobind
```

Comment:
I don't think we need `@autobind` here, do we?

Reply:
Good catch